package.json

```diff
@@ -1,70 +1,55 @@
-        "build": "webpack --config webpack.config.js",
-        "update-browsers": "npx browserslist@latest --update-db",
-        "profile": "webpack --config webpack.prod.js --profile --server"
-    },
-    "devDependencies": {
-        "autoprefixer": "^10.2.5",
-        "babel-core": "^6.26.3",
-        "babel-loader": "^8.2.2",
-        "babel-preset-env": "^1.7.0",
-        "babel-preset-stage-0": "^6.24.1",
-        "css-loader": "^5.2.4",
-        "css-minimizer-webpack-plugin": "^2.0.0",
-        "cssnano": "^5.0.2",
-        "expose-loader": "^2.0.0",
-        "file-loader": "^6.2.0",
-        "glob-all": "^3.2.1",
-        "html-webpack-plugin": "^5.3.1",
-        "image-minimizer-webpack-plugin": "^2.2.0",
-        "imagemin-gifsicle": "^7.0.0",
-        "imagemin-jpegtran": "^7.0.0",
-        "imagemin-optipng": "^8.0.0",
-        "imagemin-svgo": "^9.0.0",
-        "mini-css-extract-plugin": "^1.6.0",
-        "node-sass": "^5.0.0",
-        "path": "^0.12.7",
-        "postcss-loader": "^5.2.0",
-        "purgecss-webpack-plugin": "^4.0.3",
-        "resolve-url-loader": "^4.0.0",
-        "sass-loader": "^11.0.1",
-        "style-loader": "^2.0.0",
-        "terser-webpack-plugin": "^5.1.1",
-        "url-loader": "^4.1.1",
-        "webpack": "^5.36.2",
-        "webpack-bundle-analyzer": "^4.4.1",
-        "webpack-cli": "^4.6.0",
-        "webpack-dashboard": "^3.3.1",
-        "webpack-hook-plugin": "^1.0.7",
-        "webpack-livereload-plugin": "^3.0.1",
-        "webpack-merge": "^5.7.3",
-        "webpack-shell-plugin": "^0.5.0"
-    },
-    "dependencies": {
-        "app-root-path": "^3.0.0",
-        "audit": "0.0.6",
-        "chokidar": "^3.5.1",
-        "fix": "^0.0.3",
-        "i": "^0.3.6",
-        "jquery": "^3.6.0",
-        "normalize.css": "^8.0.1",
-        "purify-css": "^1.1.9",
-        "rootpath": "^0.1.2"
-    },
-    "consider": {},
-    "browserslist": [
-        "last 2 versions",
-        "> 1%"
-    ],
-    "try": {
-        "favicons-webpack-plugin": "^4.2.0"
-    }
+  "name": "public",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "dev": "webpack --config webpack.dev.js",
+    "watch": "webpack-dashboard -- webpack --config webpack.dev.js",
+    "build": "webpack --config webpack.config.js",
+    "update-browsers": "npx browserslist@latest --update-db",
+    "profile": "webpack --config webpack.prod.js --profile --server"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.14.3",
+    "@babel/eslint-parser": "^7.14.3",
+    "@babel/plugin-proposal-class-properties": "^7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.14.2",
+    "@babel/plugin-syntax-jsx": "^7.12.13",
+    "@babel/plugin-syntax-top-level-await": "^7.12.13",
+    "@babel/plugin-transform-react-jsx": "^7.14.3",
+    "@babel/plugin-transform-runtime": "^7.14.3",
+    "@babel/preset-env": "^7.14.2",
+    "@babel/preset-react": "^7.13.13",
```

Comment:
Babel presets to transform source files JS syntax to browser supported syntax defined at .babelrc